src/test/java/api/loans/anonymization/AnonymizeLoansImmediatelyAPITests.java

```diff
@@ -0,0 +1,223 @@
+package api.loans.anonymization;
+
+import static api.support.matchers.LoanMatchers.isAnonymized;
+import static org.hamcrest.Matchers.not;
+import static org.hamcrest.junit.MatcherAssert.assertThat;
+
+import java.net.MalformedURLException;
+import java.util.UUID;
+import java.util.concurrent.ExecutionException;
+import java.util.concurrent.TimeoutException;
+
+import api.support.builders.CheckOutByBarcodeRequestBuilder;
+import api.support.builders.LoanHistoryConfigurationBuilder;
+import org.folio.circulation.support.http.client.IndividualResource;
+import org.joda.time.DateTime;
+import org.joda.time.DateTimeUtils;
+import org.junit.Test;
+
+public class AnonymizeLoansImmediatelyAPITests extends LoanAnonymizationTests {
+
+  /**
+   * Scenario 1
+   *
+   *     Given:
+   *         An Anonymize closed loans setting of "Immediately after loan closes"
+   *         An Anonymize closed loans with associated fees/fines setting of "Immediately after fee/fine closes"
+   *         An open loan with an associated fee/fine
+   *     When the item in the loan is checked in
+   *     Then do not anonymize the loan
+   */
+  @Test
+  public void testOpenLoansWithOpenFeesAndFinesIsNotAnonymized()
+    throws InterruptedException, ExecutionException, TimeoutException, MalformedURLException {
+
+    LoanHistoryConfigurationBuilder loanHistoryConfig = new LoanHistoryConfigurationBuilder()
+      .loanCloseAnonymizeImmediately()
+      .feeFineCloseAnonymizeImmediately();
+    createConfiguration(loanHistoryConfig);
+
+    IndividualResource loanResource = loansFixture.checkOutByBarcode(new CheckOutByBarcodeRequestBuilder().forItem(item1)
+      .to(user)
+      .at(servicePoint.getId()));
+    UUID loanID = loanResource.getId();
+
+    createClosedAccountWithFeeFines(loanResource, DateTime.now());
```

Comment:
The name of this method suggests that the fees / fines account is closed. The description states that the fees and fines account is open. Which is correct?

Reply:
It's testing that loans with open fees and fines are not anonimized when the loan is closed. Otherwise, if fees and fines were closed, then it would be anonymized due to the settings.
The test have been updated.